src/main/scala/cromwell/engine/package.scala

```diff
@@ -43,19 +43,59 @@ package object engine {
     def randomId() = WorkflowId(UUID.randomUUID())
   }
 
+  object WorkflowDescriptor {
+    private def disabledMessage(readWrite: String, consequence: String) =
+      s"""$readWrite is enabled in the workflow options but Call Caching is disabled in this Cromwell instance.
+         |As a result the calls in this workflow $consequence
+       """.stripMargin
+
+    val writeDisabled = disabledMessage("Write to Cache", "WILL NOT be cached")
+    val readDisabled = disabledMessage("Read from Cache", "WILL ALL be executed")
```

Comment:
@geoffjentry This is the best compromise I could find, I couldn't convince myself to create a bunch of trait / case objects for a log message :smile: